<!--
@author:    Johann Schmidt
@date:      2019
@brief:     Launch file for starting the evaluation controller.
@todo:
-->

<launch>
    <arg name="namespace" default="tb3_"/>
    <arg name="number_of_robots" default="4"/>
    <arg name="use_settings_file" default="False"/>

    <node pkg="benchmark"
          type="evaluation_controller.py"
          name="evaluation_controller"
          output="screen"
          clear_params="true"
          if="$(eval not arg('use_settings_file'))">
        <param name="namespace" type="str" value="$(arg namespace)" />
        <param name="number_of_robots" type="str" value="$(arg number_of_robots)" />
    </node>

    <node pkg="benchmark"
          type="evaluation_controller.py"
          name="evaluation_controller"
          output="screen"
          clear_params="true"
          if="$(eval arg('use_settings_file'))">
    </node>
</launch>